### PR TITLE
Check in/68085/claim review

### PR DESCRIPTION
**Pull request by brianseek**
## Summary

Adds a review page to the travel claim flow for day-of check-in

## Related issue(s)

- _Link to ticket created in va.gov-team repo_
department-of-veterans-affairs/va.gov-team#68085

## Testing done

Adds and updates unit and e2e tests

## What areas of the site does it impact?
Day-of check-in

## Acceptance criteria
- [ ] Review page displays in travel claim flow
- [ ] terms must be accepted to file a claim
- [ ] declining acts as if the user said no to the first travel question
- [ ] clicking edit takes the user to the travel vehicle question
- [ ] trying to click Agree without checking the box causes an error

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### CODE REVIEW

File: src/applications/check-in/components/AppointmentDisplay/AppointmentAction.jsx [hunk 1]
```diff
@@ -27,7 +27,6 @@ const AppointmentAction = props => {
   const { travelPayEligible } = useTravelPayFlags(appointment);
 
   const travelSubmitted = travelPayEligible || false; // The hook returns undefined so coercing to false
-
```
Comment (brianseek) [inline src/applications/check-in/components/AppointmentDisplay/AppointmentAction.jsx:30]: ¯\_(ツ)_/¯

**Review by wullaski — DISMISSED** [review]
This all looks great! 🚀

**Review by lee-delarm6 — DISMISSED** [review]
Approved, but check my comment if you want to change the value of that hook property to be false instead of undefined 👍

**Review by lee-delarm6 — APPROVED** [review]
Looking good!

File: src/applications/check-in/components/AppointmentDisplay/AppointmentAction.jsx (hunk 1 above)
Comment (wullaski) [reply src/applications/check-in/components/AppointmentDisplay/AppointmentAction.jsx:30]: Hah that's how I interpreted the shrug so I just moved on `¯\_(ツ)_/¯`

**Review by wullaski — APPROVED** [review]
LGTM! 🚢

**Pull request merged**